examples/browserify/src/index.html

```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Polaris: Browserify example</title>
-  <link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/latest/polaris.css">
```

Comment:
This example now works without the CSS file?

Reply:
The CSS file gets imported via [app.css](https://github.com/Shopify/polaris-react/pull/1886/files#diff-fd2b3fcc106c05a6639f498803fb2db1R1) which is imported [here](https://github.com/Shopify/polaris-react/pull/1886/files#diff-97bef6d12b3ba4cbff82031326570fecR4)